### PR TITLE
Added blog post about 5k run

### PR DESCRIPTION
**Pull request by tricoder42**

### CODE REVIEW

File: tinyblog/articles/2016-03-28-5k-run.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# DjangoCon Europe 5k run
+
+> Published on 28th Mar 2016 by Tomáš Ehrlich
+
+The confrence is almost here and we're all super excited about it!
```
Comment (bmispelon) [inline tinyblog/articles/2016-03-28-5k-run.md]: Should be "conference"

**Comment by bmispelon** [comment]
Found one typo but looks good to me otherwise. Thanks!

File: tinyblog/articles/2016-03-28-5k-run.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# DjangoCon Europe 5k run
+
+> Published on 28th Mar 2016 by Tomáš Ehrlich
+
+The confrence is almost here and we're all super excited about it! 
+5 days of programming, networking and mostly fun.
+
+Let's gather together in the morning, before the conference to
```
Comment (underyx) [inline tinyblog/articles/2016-03-28-5k-run.md]: I think this needs a comma after the word 'conference'.

File: tinyblog/articles/2016-03-28-5k-run.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+get a shot of pure oxygen! There is a beatiful Margaret Island
```
Comment (underyx) [inline tinyblog/articles/2016-03-28-5k-run.md]: Feels like this sentence could be reworded a bit, how about

> The beautiful Margaret Island (Margitsziget) is only 3 km or 15 mins from the venue.

File: tinyblog/articles/2016-03-28-5k-run.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Let's gather together in the morning, before the conference to
+get a shot of pure oxygen! There is a beatiful Margaret Island 
+(Margitsziget) with even more beautiful park only 3 km or 15 mins
+from the venue. 
+
+We can meet on **31th March, at 7:00** near the **Margitsziget** 
+bus station (on the island).
+
+There is a [5 km long route](https://www.strava.com/segments/1056549) around the island,
+so we can all enjoy it at very peaceful pace.
```
Comment (underyx) [inline tinyblog/articles/2016-03-28-5k-run.md]: > There is a 5 km long route […] so we can all enjoy it

This makes it sound like there's a causal relationship between the route being there and the fact that people can run slowly :)

Not 100% sure what we tried to mean here, but how about

> There is a running track around the island. It's only 5 km long, so we can all enjoy it at a very peaceful pace and still get to the conference on time.

instead?

**Pull request merged**